README.md

```diff
@@ -157,6 +157,9 @@ So just run: `geth --datadir <datadir> init ./genesis.json`
 
 ## It is recommand to run fullnode with `--tries-verify-mode none` if you want high performance and care little about state consistency
```

Comment:
Please add some comments here: "It will run with hash trie schema by default"; Besides this, state schema should be specified when running "init genesis" in line 153.

Reply:
added